Signum.React.Extensions/Dynamic/DynamicViewClient.tsx

```diff
@@ -379,10 +380,12 @@ export function asOverrideFunction(dvo: DynamicViewOverrideEntity): (vr: ViewRep
 
 export function createDefaultDynamicView(typeName: string): Promise<DynamicViewEntity> {
   return loadNodes().then(nodes =>
-    Navigator.API.getType(typeName).then(t => DynamicViewEntity.New({

```

Comment:
Now you can use Constructor.construct(DynamicViewEntity, { 
   entityType: t, 
   viewName: "My View", 
    viewContent: JSON.stringify(nodes.NodeConstructor.createDefaultNode(getTypeInfo(typeName)))
})

Reply:
Cool!

Reply:
Done!